packages/twenty-front/src/modules/activities/components/ActivityRichTextEditor.tsx

```diff
@@ -101,7 +101,7 @@ export const ActivityRichTextEditor = ({
   const isRecordFieldReadOnly = useIsRecordFieldReadOnly({
     recordId: activityId,
     objectMetadataId: objectMetadataItemActivity.id,
-    fieldMetadataId: bodyV2FieldMetadataItemId,
+    fieldMetadataId: bodyV2FieldMetadataItem?.id ?? '',
```

Comment:
@charlesBochet fyi doing the same type of changes you did. Marie's PR was merged but it broke main